Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1039,9 +1039,13 @@ def update_remote_system_command(client, args):
 
             close_xdr_incident = argToBoolean(client._params.get("close_xdr_incident", True))
 
+            popped_status = ""
             if not close_xdr_incident and (update_args.get('status') in XSOAR_RESOLVED_STATUS_TO_XDR.values()):
-                status = update_args.pop('status')
-                demisto.debug(f"Popped {status=} from update_args, incident status won't be updated in XDR.")
+                popped_status = update_args.pop('status')
```

Comment:
Let's change the terminology back to status, and add maybe a comment for the if statement so it will be clear what it does.